### ISSUE TITLE
Payout details page doesn't work with Japanese bank details

### ISSUE THREAD

**Issue by TalAter** [sharetribe/ftw-hourly#93]
**Describe the bug** 

When entering payout details for Japanese bank accounts the submit button remains disabled even when all the details are correct.

**To Reproduce**

1. In a testing account (with a test Stripe key), go to Payout Details page.
2. Enter the following test details:
Branch name: `Akasaka`
Branch code: `000`
Bank name: `Sumitomo Mitsui Banking Corporation`
Bank code: `1100`
Bank account owner name: `John Doe`
Bank account number: `0001234`

**Expected behavior**

As those are valid details in a Stripe test environment, the submit button should become enabled.

But even though Stripe successfully returns a token, the submit button remains disabled.

![CleanShot 2020-05-11 at 11 33 26@2x](https://user-images.githubusercontent.com/926653/81541222-7320a080-937b-11ea-9320-1b8ca9b71a4a.jpg)

**Additional context and possible solution**

The problem seems to stem from the fact that FTW is sending unrecognized parameters to Stripe as can be seen here:

![CleanShot 2020-05-11 at 12 00 17@2x](https://user-images.githubusercontent.com/926653/81543706-1e7f2480-937f-11ea-884c-5239cb9fe16a.jpg)

When Stripe returns those values with the token it does not remove whitespaces from them.
FTW, on the other hand, expects those values to be "cleaned" (spaces removed) as can be seen here:
https://github.com/sharetribe/ftw-hourly/blob/ff3c18f66251ad1d9ac966cf5e2c4720157b13a6/src/components/StripeBankAccountTokenInputField/StripeBankAccountTokenInputField.js#L154

Since the result that returns from Stripe is different than expected, `valuesAreUnchanged` is greater than 0 (see line 156 in the link above) and the submit button remains disabled.

I was able to get around the problem by not cleaning up those two values:
```javascript
const changedValues = inputsNeeded.filter(
  inputType => {
    // Stripe does not "clean up" the `bankName` and `accountOwnerName` strings
    const expected = ["bankName", "accountOwnerName"].includes(inputType) ? this.state[inputType].value : cleanedString(this.state[inputType].value);
    return values[inputType] !== expected;
  }
);
```

But that seems like a hack and I am not 100% confident how this will affect other countries, and whether there are other such unrecognized parameters in other countries besides these two.

**Comment by Gnito** [comment]
@TalAter, thanks for the report! I'll check.

**Comment by Gnito** [comment]
@TalAter I ended up going to [a bit different direction](https://github.com/sharetribe/ftw-daily/pull/1287). That PR is in FTW-daily, but we'll push the change here too after some review & testing.

**Comment by Gnito** [comment]
Fix is merged to master and released in [v6.5.1](https://github.com/sharetribe/ftw-hourly/releases/tag/v6.5.1)

**Comment by TalAter** [comment]
Great.

I can also confirm that this solved a problem with Australian accounts that that used spaces in them (e.g., BSB: `110 000`, Bank account number: `000 123 456`).
